### PR TITLE
Updated the Learner docs page

### PR DESCRIPTION
**Pull request by radinamatic**
## Summary

Current screenshots and new/changed features.

### CODE REVIEW

File: docs/user/learner.rst [hunk 1]
```diff
@@ -4,24 +4,27 @@ Learner
 Accessing Kolibri
 -----------------
 
-To log in to **Kolibri** follow these steps:
+To sign in to **Kolibri** and start learning follow these steps:
 
-#. Click the **Log In** icon in the main menu (left or bottom) to open the login window.
+#. Click the **Sign in** icon either in the upper right corner, or in the main menu (left or bottom) to open the sign-in page.
```
Comment (DXCanas) [inline docs/user/learner.rst]: I think we should specify that you would only ever have to sign in using the menus if you decided to "Access as Guest" - currently, the default behavior of the app when no one is signed on is to take you to the `user` plugin.

File: docs/user/learner.rst [hunk 2]
```diff
@@ -4,24 +4,27 @@ Learner
...
 
-To logout from Kolibri follow these steps:
+.. image:: img/update_profile.png
+	:alt: view and edit profile
 
-#. Click the icon with your initial in the main menu.
-#. Select **Log Out** option.
+To logout from Kolibri you can either:
+
+* Click the user icon in the upper right corner, and select **Sign out** option.
```
Comment (DXCanas) [inline docs/user/learner.rst]: No comma needed here

File: docs/user/learner.rst [hunk 3]
```diff
@@ -41,27 +44,43 @@ Learn
 
 Each time you login into **Kolibri**, the first thing you will see is the **Learn** page. Here you will find learning topics and materials related to what you were doing the last time you used Kolibri, or those recommended by your teachers and coaches.
```
Comment (DXCanas) [inline docs/user/learner.rst]: Only true if signed in

File: docs/user/learner.rst [hunk 4]
```diff
@@ -41,27 +44,43 @@ Learn
...
 
+The content you see in the **Learn** page will be different for each channel. Change the channel to explore the resto of content you can use in Kolibri.
```
Comment (DXCanas) [inline docs/user/learner.rst]: > resto

File: docs/user/learner.rst [hunk 5]
```diff
@@ -41,27 +44,43 @@ Learn
...
+**Topics** tab offers you the option to navigate through the complete set of learning topics and materials available in Kolibri, for the currently selected channel. Use it as you wish, or according to indications from your teachers and coaches. 
 
-To go back (up one level from where you are currently), click the next to last link in the breadcrumb. 
+.. image:: img/explore.png
+	:alt: topics tab
+
+Navigate Kolibri topics with breadcrumb links
+*********************************************
+
+When you are browsing a topic in Kolibri, the **Breadcrumb** links indicate all its parent topics. To go back (up one level from where you are currently), click the next to last link in the breadcrumb.
```
Comment (DXCanas) [inline docs/user/learner.rst]: > all its parent topics. 

A bit confusing and possibly innacurate.  I would go with something like:

"... the **Breadcrumbs** indicate the current topic and every topic visited prior."

>To go back (up one level from where you are currently), click the next to last link in the breadcrumb.

I think we should refrain from using the "up" metaphor, seeing as the breadcrumbs are horizontally oriented.

"Next to last" can be confusing as well. Maybe "First link from the left of the current page's title"? Not sure, but could be refined.

File: docs/user/learner.rst [hunk 6]
```diff
@@ -41,27 +44,43 @@ Learn
...
+
+Navigate Kolibri topics with breadcrumb links
+*********************************************
+
+When you are browsing a topic in Kolibri, the **Breadcrumb** links indicate all its parent topics. To go back (up one level from where you are currently), click the next to last link in the breadcrumb. 
 
 .. image:: img/breadcrumbs.png
 	:alt: breadcrumb links
 
+When you are viewing a video or doing an exercise, the **Breadcrumb** links are not visible anymore. To go back to the parent topic from where you are currently, and see the **Breadcrumb** links again, click the left arrow button above the title.
```
Comment (DXCanas) [inline docs/user/learner.rst]: A bit redundant/wordy. I would cut out the non-defining subordinate clause and do a little cleanup:

"When you visit a video exercise, the **Breadcrumb** links are no longer visible. To go back to the last topic you visited, click on the left arrow button above the title."

P.S. [I had to google what a statement in between commas was to find out that it was called a non-defining subordinate clause](http://grammar.yourdictionary.com/punctuation/comma-rules.html)

File: docs/user/learner.rst [hunk 7]
```diff
@@ -74,10 +93,30 @@ If you are looking for a specific subject, topic, or term, use the **Search** fe
...
+
+Independent of the required action (writing an answer yourself or choosing one of the options), these are the steps to follow.
+
+#. **Read the question carefully**.
+#. When you are ready to submit the answer, click the **Check answer** button.
+
+   * If the answer is correct and a checkmark appears, click the **Next question** button to proceed.
+   * If the answer is incorrect, click the **Get a hint** button, read the suggestions, and try to answer again.
+
+#. Once you have achieved 5 correct answers in a row, click the **Next item** button, to continue learning with the rest of the material in that topic.
```
Comment (DXCanas) [inline docs/user/learner.rst]: I believe this is only true if that was the setting defined by the content curator. @rtibbles?

File: docs/user/learner.rst [hunk 8]
```diff
@@ -86,15 +125,14 @@ To play videos in **Kolibri** you have several available control buttons at the
...
+* Time tracker indicator with progress bar
 * Video duration indicator
-* Mute sound button
 * Volume scrollbar
 * Playback speed selector
 * Fullscreen button
 
 
-Use the **Download Media** button below the video player to download the video file to your computer.
+Use the **Download content** button below the video player to download the video and thumbnail files to your computer. On some videos you will have an option to download videos in high and low resolution.
```
Comment (DXCanas) [inline docs/user/learner.rst]: > On some videos you will have an option to download videos in high and low resolution

would require a comma:
"On some videos, you will have an option to download videos in high and low resolutions"

Personally, I'd rather do:
"Some videos will even provide multiple resolution options".

File: docs/user/learner.rst (hunk 1 above)
Comment (radinamatic) [reply docs/user/learner.rst]: Added!

File: docs/user/learner.rst (hunk 3 above)
Comment (radinamatic) [reply docs/user/learner.rst]: Added.

File: docs/user/learner.rst (hunk 5 above)
Comment (radinamatic) [reply docs/user/learner.rst]: Good point about the "up" metaphor! :+1: 
`every topic visited prior` sounded a bit too strong, let me know if this latest edit sounds clearer.

File: docs/user/learner.rst (hunk 6 above)
Comment (radinamatic) [reply docs/user/learner.rst]: :bowing_woman: for this attention to language!

File: docs/user/learner.rst (hunk 7 above)
Comment (radinamatic) [reply docs/user/learner.rst]: Edited!

**Comment by radinamatic** [comment]
Excellent feedback in this review, @DXCanas! Let me know if you have other suggestions.

**Comment by indirectlylit** [comment]
merging into `0.4.x` rather than `develop`

**Pull request merged**